packages/headless/src/features/search-and-folding/build-search-request.ts

```diff
@@ -0,0 +1,68 @@
+import {getVisitorID} from '../../api/analytics/analytics';
```

Comment:
Does the directory name need to mention `folding`?

Reply:
Similar to my comment below: I really wanna emphasize to future maintainer that it's not a "generic base build search request thingy", it's about folding and search specifically.